templates/kcc/product/service_mapping.yaml.erb

```diff
@@ -0,0 +1,91 @@
+<%- # the license inside this block applies to this file
```

Comment:
fwiw, over in Ansible land, I've found it easier to build out Ruby dictionaries + then just call `.to_yaml` on those data structures when I'm trying to build YAML.

I always like having Ruby deal with YAML data formatting rather than me.

Reply:
I think my preference is to keep it as a template. I think it'll be easier for occasional contributors, and more consistent with other Terraform-provider based providers. It's also very clear how template changes will affect what downstream files, which I'd be afraid of losing in the transition.